### ISSUE TITLE
About normalization

### ISSUE THREAD

**Issue by sian-chen** [ditschuk/pytorch-tsmixer#2]
Hi,

I'm an author of TSMixer.
Thanks for sharing the implementation.
I'd like to inform you that we have used LayerNorm rather than BatchNorm for better stability in handling non-stationary time series.
Also, though there is no significant difference in our experiments, I normalized both the time and feature dimensions (2D norm) rather than time dimension only (1D norm).

Best,
Si-An

**Comment by ditschuk** [comment]
Hi Si-An,

thanks a lot for the great paper and for providing valuable feedback on my implementation!

I tried to adapt the code to your feedback in this PR #3 :
* I adapted the API so that users can choose between layer and batch norm. 
* The batch norm should now normalize over time and feature dimensions. (I think the `keras` implementation of BatchNorm is more flexible than that of `pytorch`, so I had to hack it together a bit)

Feel free to comment on the changes in the PR or this thread.
(Otherwise, I will merge the PR in a few days.)

Best,
Konstantin
